images/ap-puns/service.nix

```diff
@@ -2,13 +2,65 @@
 with builtins;
 let
   puns = [
-    "Pretty Fly for a WiFi"
-    "Drop it Like It's Hot Spot"
+    # LAN
     "LAN Before Time"
+    "Life in the fast LAN"
+    "The Promised LAN"
+    "House LANnister"
+    "Silence of the LAN"
+    "No LAN for the Wicked"
+    "LAN of Milk and Honey"
+    "The LAN of the Free"
+    "Winter WonderLAN"
+    "Wu Tang LAN"
+    "Iron LAN"
+    "SpiderLAN"
+    
+    # WiFi
     "The WiFi of Khan"
-    "It Burns When IP"
-    "New England Clam Router"
+    "Pretty Fly for a WiFi"
     "Hide Your Kids Hide Your WiFi"
+    "Tell my WiFi I love her"
+    "No more Mr Wi-Fi"
+    "I now prononouce you man and wifi"
+    "They are taking the hobbits to WiFisengard!"
```

Comment:
1 out of Fi, this is shameful

Reply:
One of the few originals in the patch set :1st_place_medal: